### PR TITLE
Adding workouts works

### LINKED ISSUE #68: User profile - Add new workout

### LINKED ISSUE #118: Fix Up/Down counters for Reps

### PR DESCRIPTION
**Pull request by Alexander-Perry**
Still todo: Up/down buttons for the reps.
Will do as a separate pull request after sorting

### CODE REVIEW

**Comment by Alexander-Perry** [comment]
* Up/Down Counters work
* Categories display on workouts page
* If existing workout, updates the reps accordingly
* Fixed 'reps' not adding the correct data when saving workouts
* Can now add workouts to profile. 
* fixed up indentation and added a few comments

File: controllers/api/workoutRoutes.js [hunk 1]
```diff
@@ -1,132 +1,40 @@
...
-    // ROUTES NEEDED:
-    // (a) create new workout
-    // (b) get one workout
-    // (c) get all workouts
-    // (d) delete workout
 ////////////////////////////////////////////////////////
-
+// Future Work: ROUTES NEEDED:
 // (a) create new workout
-router.post('/', withAuth, async (req, res) => {
```
Comment (Alexander-Perry) [inline controllers/api/workoutRoutes.js:15]: These are unused. These routes are handled by homeRoutes.js

File: public/js/workout.js [hunk 2]
```diff
@@ -1,55 +1,39 @@
-
```
Comment (Alexander-Perry) [inline public/js/workout.js:1]: Functionality is moved into a single event listener (bubbling)

File: views/workouts.handlebars [hunk 3]
```diff
@@ -1,31 +1,29 @@
 {{#if workouts.length}}
```
Comment (Alexander-Perry) [inline views/workouts.handlebars:1]: It looks like more is changed here than actually is... 
a few of the id and data tags were adjusted - closing /each moved up a few lines to fix the event listener issue

**Review by WollemiPines — APPROVED** [review]

**Pull request merged**